pkg/utils/util.go

```diff
@@ -250,3 +250,17 @@ func CheckBackplaneVersion(cmd *cobra.Command) {
 
 	logger.WithField("Current version", info.Version).WithField("Latest version", latestVersion).Warn("Your Backplane CLI is not up to date. Please run the command 'ocm backplane upgrade' to upgrade to the latest version")
 }
+
+var ShellChecker ShellCheckerInterface = DefaultShellChecker{}
+
+type ShellCheckerInterface interface {
+	IsValidShell(shellPath string) bool
+}
+
+type DefaultShellChecker struct{}
+
+// Helper function to check if a shell is valid
+func (checker DefaultShellChecker) IsValidShell(shellPath string) bool {
+	_, err := os.Stat(shellPath)
+	return err == nil
+}
```

Comment:
Not a blocker.. 
great to see that func move to interface and mock via the tests :) 

I feel that it shouldn't be part of `pkg/util/util.go` may be `pkg/util/shell.go`